command-acceptor/app/controller/implementations/RequestAnalyser.py

```diff
@@ -0,0 +1,62 @@
+from flask import request
+#from controller import interfaces
+import app.controller.interfaces as interfaces
+from app.models.User import User
+from app.models.Chat import Chat
+from app.models.ErrorMessage import ErrorMessage
+from app.models.AcceptedMessage import AcceptedMessage
+from jsonobject import DictProperty
+
+
+class RequestAnalyser(interfaces.RequestAnalyser):
+    def __init__(self, request_in: request) -> None:
```

Comment:
method ```__init__``` (aka constructor) shouldn't do a heavy work at all. 
I would recommend to do with work in a separate method ```process```.

Reply:
Done